services/movie_session.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int,
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
+    movie_sessions = MovieSession.objects.all()
+    if session_date:
+        movie_sessions = movie_sessions.filter(
+            show_time__date=datetime.strptime(session_date, "%Y-%m-%d")
```

Comment:
Or you could more simply filter like so `show_time__date=session_date` django will take care about type conversion